### DISCUSSION TITLE
Issue form with better field support

### DISCUSSION THREAD

**Discussion by Timmmy-nlb** [community/community#4317] [Projects and Issues]
I would like to see better field support.
We are now working with input between placeholders to process the inputs in workflows.
But even this is unfortunately always error-prone due to the user component.
A better field content transfer would improve this.

**Comment by lukehefson** [comment]
Hi @Timmmy-nlb 👋

I’m a little unsure what your current workflow looks like. Could you perhaps provide us with some screenshots of what you’re doing and a little more info around what you’d like to see?

**Reply by Timmmy-nlb** [reply]
Hello @lukehefson,
we have build few workflow for 'low tech' administration in our enterprise organisations. 
We deny org owner rights and use workflows for automatic jobs like create new repos or invite new user.

For thats reasons we need imputs form the users in issues. We 'create' a form with enclosing sperators (stars in our case): 
`Username:  *sample* `
In workflow we disassemble the issue body and try to get the input.

But they are only humens and make mistakes ;)
They delete the stars, insert white space and more. 
This makes the extraction process problematic und unsecure.

My hope is to get fields which I can directly address, without a sperator crutch.
